arcgis-ios-sdk-samples/Display information/Add graphics with renderer/README.md

```diff
@@ -10,7 +10,7 @@ A renderer allows you to change the style of all graphics in an overlay by only
 
 ## How to use the sample
 
-Run the sample and view graphics for points, lines, and polygons, which are stylized using renderers.
+Run the sample and view graphics for points, lines, polygons, and shapes with curved segments which are stylized using renderers.
```

Comment:
```suggestion
Pan and zoom on the map to view graphics for points, lines, polygons, and shapes with curved segments which are stylized using renderers.
```
I think it's been a while since we've updated this sample and README. A slight change would keep this up to date with our current guidelines.